tests/playwright/README.md

```diff
@@ -1,123 +1,47 @@
 # Podman Desktop Playwright Tests
 
-Testing Framework dedicated to a Podman Desktop and its extensions.
+This document contains information on how to use the E2E testing framework of Podman Desktop from an external project.
+This is particularly useful if you want to execute the tests from a Podman Desktop extension or you want to develop your own.
 
-## How to develop using locally built @podman-desktop/tests-playwright
+## Usage of @podman-desktop/tests-playwright locally
```

Comment:
I would not really change this section. It is about how you can locally incorporate some changes into `tests-playwright` package, build it locally and test it locally. Maybe it would be good to just change the order with next section.

Possible outcome:
the first section -> usage of tests-playwright when you develop the tests in podman-desktop (basically we are referencing and importing the modules from actual or parent folder. no need to use the npm package)
Second section -> Usage of tests-playwright in your external repository (external to podman-desktop)
Third section -> Local development of the npm package (tests-playwright) and testing it in the external repo